rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/CompactView.java

```diff
@@ -459,6 +451,9 @@ public void onBrowserEvent(Event event) {
                     String lockItemImageResource = RMImages.instance.node_add_16_locked().getSafeUri().asString();
                     String unlockItemImageResource = RMImages.instance.node_add_16().getSafeUri().asString();
                     String deployItemImageResource = RMImages.instance.nodesource_deployed_16().getSafeUri().asString();
+                    String undeployItemImageResource = RMImages.instance.nodesource_undeployed_16()
+                                                                        .getSafeUri()
+                                                                        .asString();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)